src/edit/options.js

```diff
@@ -132,6 +132,7 @@ export function defineOptions(CodeMirror) {
     }
     cm.display.input.readOnlyChanged(val)
   })
+  option("screenReaderLabel", 'CodeMirror text input')
```

Comment:
I think it'd make sense to default this to null (and have the input implementations avoid setting the attribute when the option is null). No need to spam screen reader users with text when we don't have anything relevant to add.

Also, it'd be good if this option could be changed dynamically. Instead of integrating the setting of the attribute in the input object creation, maybe make it a method that the editor calls on init, and then again when this option is changed.

Reply:
OK. Got it.